### DISCUSSION TITLE
Integration test with `pack` CLI.

### DISCUSSION THREAD

**Discussion by matejvasek** [containers/podman#20339] [General]
Hi all,

podman does provide compatibility Docker API,
it mostly works alright, still occasionally  I see compatibility issues with [pack](https://github.com/buildpacks/pack) CLI.

My question is: is it conceivable to add some integration test running [pack](https://github.com/buildpacks/pack) command against podman?

I know this is not standard requirement since every docker API consumer could ask for same thing.

**Comment by Luap99** [comment]
As in adding your pack tests to our CI infrastructure?
Given we likely would not be able to understand test failures and we would not want to block merges on third party tests I don't think it would add much value for us.

I think it would be better for us if you could run your integration tests using https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/ on some nightly job and then report bugs if you notice any regressions. Then we can fix or revert the change which caused the problem.

**Comment by matejvasek** [comment]
@Luap99 Is that some Fedora specific test?

**Reply by Luap99** [reply]
https://copr.fedorainfracloud.org/coprs/rhcontainerbot/podman-next/ is just a repository with the latest podman and deps build from the main branches. How you run tests is totally up to you it is just that these repos are an easy way to install the latest packages so you can notice regression before they end up in the final release.
